identity-enabler/verifier-mobile-app/src/pages/DevInfo.svelte

```diff
@@ -15,7 +16,8 @@
         try {
             code = await getMarkdownContent(`${TUTORIAL_BASE_URL}/${page}.md`);
         } catch (err) {
-            error.set("Error getting markdown file. Please try again.");
```

Comment:
There was no import for `error`, so this would have caused an error, but it is redundant anyway since the `error` store is not used anywhere else.